preprocessors/ner/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
```

Comment:
Do you specifically need JRE 8 for this to work??

Reply:
This is mentioned on the Stanford ner website:
`Either make sure you have or get [Java 8](http://java.com/) or consider running an earlier version of the software (versions through 3.4.1 support Java 6 and 7).. (https://nlp.stanford.edu/software/CRF-NER.html#Download)`

For what I understand version 8 is the latest that would work. right?